package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "knex-mysql-deadlock",
+  "version": "1.0.0",
+  "description": "A custom knex mysql dialect, which does deadlock retries",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Mindflash/knex-mysql-deadlock.git"
+  },
+  "author": "mindflash",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Mindflash/knex-mysql-deadlock/issues"
+  },
+  "homepage": "https://github.com/Mindflash/knex-mysql-deadlock#readme",
+  "dependencies": {
+    "knex": "^0.12.6"
```

Comment:
I'm not super against making node-mysql a peerDep, I'm just trying to understand what's the point. As we have `knex` as a peerDep, it will have to have node-mysql installed no matter what, and with that said - duplicating node-mysql as a peerDep in this module doesn't make any sense really from my pov.